cmd/web-handlers.go

```diff
@@ -21,6 +21,11 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/dustin/go-humanize"
```

Comment:
imports should be organized little differently follow this -
 https://github.com/golang/go/wiki/CodeReviewComments#imports